### DISCUSSION TITLE
Small tutorial how to setup a nice looking Powershell tutorial!

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [felixse/FluentTerminal#962] [Show and tell]
I just wrote a small tutorial/guide on how to setup a nice looking powershell 7 terminal, that uses Fluent Terminal..

[Bedazzling Windows Terminal](https://anderssonpeter.github.io/Bedazzling-Windows-Terminal/)

If your setup differs or you have any ideas how to improve please comment :)

**Comment by hanskokx** [comment]
This looks great! Thanks for contributing! I use Hack Nerd Font with all the same settings, mostly.  I have my own, custom oh-my-posh theme, though:

![image](https://user-images.githubusercontent.com/1911919/134030716-6b1185e2-409b-4847-a7cb-468dd077aae4.png)

**Reply by AnderssonPeter** [reply]
Nice theme even if your spotify song seems overkill :p

**Reply by hanskokx** [reply]
> Nice theme even if your spotify song seems overkill :p

What do you have against my Queen, Taylor Swift?
